### PR TITLE
[BE] Fix all B022 `useless-contextlib-suppress`

### PR DESCRIPTION
**Pull request by justinchuby**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #100336
* #100342
* #100341
* #100339
* #100338
* #100337
* __->__ #100335

No arguments passed to contextlib.suppress. No exceptions will be suppressed and therefore this context manager is redundant

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -30,7 +30,6 @@ ignore = [
     "B007", "B008", "B017",
     "B018", # Useless expression
     "B019", "B020",
-    "B022", # Allow empty context manager
```
Comment (Skylion007) [inline pyproject.toml:33]: remove the ignore from .flake8 file as well.

Comment (justinchuby) [reply pyproject.toml:33]: There was not an entry in flake8 for some reason.

Comment (Skylion007) [reply pyproject.toml:33]: Interesting...

**Comment by Skylion007** [comment]
BTW, if you are planning on opening a series of PRs and removing a lot flake8 ignores, focus on the 'B' codes ones as a lot of those are currently hiding bad / buggy behavior.

File: pyproject.toml (hunk 1 above)
Comment (justinchuby) [reply pyproject.toml:33]: Potentially the bugbear version we use is an older one

**Comment by justinchuby** [comment]
> BTW, if you are planning on opening a series of PRs and removing a lot flake8 ignores, focus on the 'B' codes ones as a lot of those are currently hiding bad / buggy behavior.

SG. I went over all of them and did the ones with autofix available.

**Comment by justinchuby** [comment]
Also https://github.com/pytorch/pytorch/pull/100342

**Pull request closed without merging**